config.json

```diff
@@ -18,9 +18,6 @@
   "test_runner": {
     "average_run_time": 6
   },
-  "approaches": {
-    "snippet_extension": "cr"
-  },
```

Comment:
`bin/configlet` did this when I ran `create`. Is this section not actually used?

Reply:
It is not.
It is filled as an option in the docs but isn't yet implemented